sdk/daml-lf/encoder/src/main/scala/com/digitalasset/daml/lf/archive/testing/DamlLfEncoder.scala

```diff
@@ -47,7 +44,7 @@ private[daml] object DamlLfEncoder extends App {
         )
 
       makeDar(
-        readSources(appArgs.inputFiles),
+        readSources(appArgs.inputFiles.reverse),
```

Comment:
Why reverse? Maybe add a comment?

Reply:
I change the type of `inputFiles` from `List` to `Queue`, so no need to reverse.